### PR TITLE
Feat: CheckItinerary params

### PR DESCRIPTION
**Pull request by JoseDuSV**
Summary: Replaced mock values with real params for CheckItinerary query.
Which means that result is matching value from results list.

### CODE REVIEW

File: apps/core/scenes/resultDetail/ResultDetail.js [hunk 1]
```diff
@@ -25,9 +28,12 @@ export default class ResultDetail extends React.Component<Props> {
         `}
         variables={{
           input: {
-            bookingToken: this.props.bookingToken,
+            bookingToken,
             bags: 0, // @TODO - use real data from search context
-            passengers: { adults: 1 }, // @TODO - use real data
+            passengers: {
+              adults: adults ? parseInt(adults, 10) : 1,
```
Comment (JoseDuSV) [inline apps/core/scenes/resultDetail/ResultDetail.js]: I'm not that happy about this part. Mainly because we are mixing values from url and search context in few places. I want to do some refactoring in different PR and use SearchContext as main source and parser of parameters for Search/Results/ResultDetail . I just need to sync with @aalises  because he will probably need something similar for search input above web results.

Comment (tbergq) [reply apps/core/scenes/resultDetail/ResultDetail.js]: I guess this is string | number because it is passed differently from web and mobile?

How about doing the parseInt on the parent, so the type here can be just number?

Comment (JoseDuSV) [reply apps/core/scenes/resultDetail/ResultDetail.js]: @tbergq Good idea. Moved parsing to parent for web version (where params are string) which makes it a bit cleaner. Thanks

File: apps/web/pages/result-detail.js [hunk 2]
```diff
@@ -9,10 +9,16 @@ import { withPageRouter } from '../components/withPageRouter';
 import Layout from '../components/Layout';
 
 const resultDetail = ({ router }) => {
+  const { bookingToken, adults, infants } = router.query;
+  const params = {
+    bookingToken,
+    adults: adults ? parseInt(adults, 10) : 1,
+    infants: infants ? parseInt(infants, 10) : 0,
+  };
```
Comment (tbergq) [inline apps/web/pages/result-detail.js]: What about

```
const params = {
    bookingToken,
    adults: parseInt(adults ?? 1, 10),
    infants: parseInt(infants ?? 0, 10),
  };
```

Comment (JoseDuSV) [reply apps/web/pages/result-detail.js]: 👍 Definitely better. Updated
